src/Symfony/Component/PropertyInfo/Extractor/PhpStanExtractor.php

```diff
@@ -206,7 +207,7 @@ public function getType(string $class, string $property, array $context = []): ?
         $types = [];
 
         foreach ($docNode->getTagsByName($tag) as $tagDocNode) {
-            if ($tagDocNode->value instanceof InvalidTagValueNode) {
+            if ($tagDocNode->value instanceof InvalidTagValueNode || $tagDocNode->value instanceof TypelessParamTagValueNode) {
```

Comment:
Maybe we can make it explicit instead?
```suggestion
            if (!$tagDocNode->value instanceof ParamTagValueNode && !$tagDocNode->value instanceof ReturnTagValueNode && !$tagDocNode->value instanceof VarTagValueNode) {
```

Reply:
I have updated it to be more explicit.